appservice/src/intTest/groovy/nu/yona/server/BasicBuddyTest.groovy

```diff
@@ -76,11 +76,13 @@ class BasicBuddyTest extends AbstractAppServiceIntegrationTest
 
 		then:
 		response.status == 200
-		response.responseData._embedded.buddyConnectRequestMessages[0].user.firstName == "Richard"
```

Comment:
Not by accident. I felt checking the nickname is sufficient. But when I think about it again, I feel we must also check the embedded user object, so I've re-added it.